CMakeLists.txt

```diff
@@ -24,8 +24,8 @@ MESSAGE(STATUS "====================================================")
 # Set the name of the project and target:
 SET(TARGET "aspect")
 
-FILE(GLOB_RECURSE TARGET_SRC  "source/*.cc" "include/*.h")
-INCLUDE_DIRECTORIES(include ${CMAKE_BINARY_DIR}/include)
+FILE(GLOB_RECURSE TARGET_SRC  "source/*.cc" "unit_tests/*.cc" "include/*.h" "contrib/catch/catch.hpp")
```

Comment:
Do we need the header files here? I wonder whether `include/*.h` actually expands to anything, but it's something I don't know one way or the other anyway.